metadata/boundaries/school_district_boundaries.md

```diff
@@ -0,0 +1,77 @@
+# Title
+
+Utah School District Boundaries
+
+## ID
+
+fb17c1bb-954a-4a3b-997a-7024cae04fbc
+
+## Brief Summary
+
+Polygon dataset representing more than 40 school districts in Utah.
```

Comment:
```suggestion
Polygon dataset representing the more than 40 school districts in Utah.
```
"The" implies it is an exhaustive dataset.

Reply:
I feel adding the "the" makes it sound awkward. I changed it to "Polygon dataset representing geographic boundaries of all school districts in Utah." Thoughts?